### PR TITLE
updating status pill

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Row View Before:
image

Row View After:
image

Card View Before:
image

Card View After:
image

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
@scott062 I choose you - review my PR, Scott 😈

File: src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx [hunk 1]
```diff
@@ -31,6 +32,8 @@ const AgendaItemCard = props => {
   const createAI = () => { console.log('placeholder create AI'); };
   // eslint-disable-next-line no-console
   const editAI = () => { console.log('placeholder create AI'); };
+  const pillColor = pillColors[get(agenda, 'status') || 'Default'];
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx:35]: Could make this a variable to be used here (line 35 and line 63) if you want

Comment (elizabeth-jimenez) [reply src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx]: pass

just kidding - updated!

**Comment by scott062** [comment]
Screen Shot 2022-01-21 at 1 33 46 PM
On Edge 😿

**Review by scott062 — APPROVED** [review]

**Pull request merged**